packages/create-vite/src/index.ts

```diff
@@ -553,6 +578,27 @@ async function init() {
 
   pkg.name = packageName
 
+  if (useRolldownVite) {
+    // renovate: datasource=npm depName=rolldown-vite
+    const rolldownViteVersion = '7.1.12'
+    const pkgVersion = `npm:rolldown-vite@${rolldownViteVersion}`
```

Comment:
```suggestion
    const pkgVersion = `npm:rolldown-vite@^${rolldownViteVersion}`
```

Should we use a caret here so it's not pinned? (I think it works but haven't tested)

Reply:
I'll prefer to keep without it as rolldown-vite doesn't necessarily follow semver.

Reply:
Like this section?
https://vite.dev/guide/rolldown.html#versioning-policy

Reply:
Yeah I mean the docs mentioning `"npm:rolldown-vite@latest"`, so it's also not pinned to a specific version.